nuxeo-retention/pom.xml

```diff
@@ -88,5 +88,15 @@
       <artifactId>nuxeo-platform-default-config</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-search-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
For information, this is a transitive dependency of nuxeo-platform-query-api which is already in the compile scope.

Reply:
Now needed for `CoreSearchFeatrure` -> https://github.com/nuxeo/nuxeo-retention/pull/184/files#diff-a182f82200879f685b5410d4e93c73106a54309d8456f97bb330070d35d72de2R32

Reply:
`CoreSearchFeature` is in nuxeo-core-test module 🤔.

Reply:
My bad, it is for the `@Deploy("org.nuxeo.ecm.platform.search.core")` -> https://github.com/nuxeo/nuxeo-retention/pull/184/files#diff-a182f82200879f685b5410d4e93c73106a54309d8456f97bb330070d35d72de2R37

Reply:
:warning: `nuxeo-core-search` != `nuxeo-search-core`